app/controllers/buyers/service_contracts_controller.rb

```diff
@@ -37,6 +38,7 @@ def index
 
     @service_contracts = current_user.accessible_service_contracts
```

Comment:
@thalesmiguel 

> By the way, where are those `provided_by` methods used? I ask because I don't see those being used in this PR

They are used indirectly from here, this `accessible_service_contracts` method.
The chain then goes [here](https://github.com/3scale/porta/blob/885787f3e812748bee434436afc281d068fad9c4/app/models/user.rb#L219) and [here](https://github.com/3scale/porta/blob/885787f3e812748bee434436afc281d068fad9c4/app/models/account/provider_methods.rb#L201) and this is where `provided_by` is used. Just that originally it was the parent's class method, and I added one for the subclass to shrink the scope.